docs/reference/generated/alert-dialog-description.json

```diff
@@ -3,11 +3,11 @@
   "description": "A paragraph with additional information about the alert dialog.\nRenders a `<p>` element.",
   "props": {
     "className": {
-      "type": "string | (state) => string",
+      "type": "string | ((state: unknown) => string)",
       "description": "CSS class applied to the element, or a function that\nreturns a class based on the component’s state."
     },
     "render": {
-      "type": "React.ReactElement | (props, state) => React.ReactElement",
+      "type": "React.ReactElement | ((props: unknown, state: unknown) => React.ReactElement)",
```

Comment:
We probably need to specify the state & props for each component.

Reply:
Yeah, this doesn't look right. These aren't really the props of a component but attributes to be spread on the rendered HTML element. I'll try to display in a better way